File: Course_01/C Language/EstruturaSequencial/idades.c
#include <stdio.h>
#include <string.h>

void clean_entry() {
    char c;
    while ((c = getchar()) != '\n' && c != EOF) {}
}
void read_text(char *buffer, int length) {
    fgets(buffer, length, stdin);
    strtok(buffer, "\n");
}


int main()
{
    char nome1[50], nome2[50];
    int idade1, idade2;
    double media;

    printf("Dados da primeira pessoa:%\n");
    printf("Nome: ");
    read_text(nome1, 50);
    printf("Idade: ");
    scanf("%d", &idade1);

    printf("Dados da segunda pessoa:%\n");
    printf("Nome: ");
    clean_entry();
    read_text(nome2, 50);
    printf("Idade: ");
    scanf("%d", &idade2);

    media = ((double) idade1 + (double) idade2) / 2;

    printf("%\n A idade media de %s e %s e de %.2lf anos", nome1, nome2, media);

    return 0;
}

File: Course_01/Java Language/Estrutura sequencial/Idades.java
package idades;

import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		
		Scanner sc = new Scanner(System.in);
		
		String nome1, nome2;
		int idade1, idade2;
		double media;
		
		System.out.println("Dados da primeira pessoa:");
		System.out.print("Nome: ");
		nome1 = sc.nextLine();
		System.out.println("Idade: ");
		idade1 = sc.nextInt();
		
		System.out.println("Dados da segunda pessoa: ");
		System.out.println("Nome: ");
		sc.nextLine();
		nome2 = sc.nextLine();
		System.out.println("Idade: ");
		idade2 = sc.nextInt();
		
		media = (double) (idade1 + idade2) / 2;
		
		System.out.println("A idade media de " + nome1 + " e " + nome2 + " e de " + String.format("%.1f", media) + " anos");
		
		sc.close();
	}

}